File: ins.js
let arr = [1, 5, 3, 9, 2];
let sortedArray = sort(arr);
console.log(sortedArray);

function sort(arr) {
  for (let i = 1; i < arr.length; i++) {
    let j = i - 1;
    let key = arr[i];

    while (j >= 0 && key < arr[j]) {
      arr[j + 1] = arr[j];
      j--;
    }
    arr[j + 1] = key;
  }
  console.log(arr);
  return arr;
}

File: ins.py
def sort(arr):
    for right in range(1, len(arr)):
        left = right - 1
        elem = arr[right]

        while left >= 0 and arr[left] > elem:
            arr[left + 1] = arr[left]
            left -= 1
        arr[left + 1] = elem

    return arr


arr = [1, 2, 6, 3, 7, 5, 9]
print(sort(arr))
